docs/content/linting.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Linting
+description: Primer React offers an ESLint plugin to enforce best practices and fix common problems. 
+---
+
+<Note variant="warning">
```

Comment:
🤔 do we treat all `.md` files as `.mdx`?

Reply:
Yup. Here's where that's configured: https://github.com/primer/doctocat/blob/master/theme/gatsby-config.js#L21